include/deal.II/base/array_view.h

```diff
@@ -643,9 +715,10 @@ make_array_view (Table<2,ElementType> &table)
  * initializing an ArrayView object with a pointer to the first element of the
  * given table, and the number of table entries as the length of the view.
  *
- * This function is used for @p const references to objects of Table type
- * because they contain immutable elements. Consequently, the return type of
- * this function is a view to a set of @p const objects.
+ * This function is used for non-@p const references to objects of
```

Comment:
This change seems wrong here -- it is in the comment of the version for `Table`.